search/search_indexes.py

```diff
@@ -30,6 +31,12 @@ def index_queryset(self, using=None):
         """Records to check when updating entire index."""
         return self.get_model().objects.all()
 
+    def prepare_text(self, obj):  # pylint: disable=no-self-use
```

Comment:
@Ferdi: Yes. If we do that, then only what is described in `prepare_text` will be searched when someone types in the search box. Everything else will be a separate filter.

@carsongee: This is a feature of Haystack, just like the `clean_fieldname` form override in Django. However, unlike Django's "clean" methods, it does not use something like `self.cleaned_data`, so there's just no "self" use.